services/fncsgossbridge/service/fncs_goss_bridge.py

```diff
@@ -593,7 +632,7 @@ def _done_with_time_step(current_time):
         
             
 def _register_with_goss(sim_id,username,password,goss_server='localhost', 
-                      stomp_port='61613',):
+                      stomp_port='61613', sim_durration=86400):
```

Comment:
what is this magic duration 86400?

Reply:
one day

Reply:
Should this be a constant at the top of the file or somewhere else to reference it...you use it a couple of different places in the file?